src/common/Gateway.sol

```diff
@@ -275,8 +273,14 @@ contract Gateway is Auth, Recoverable, IGateway {
             }
         }
 
-        batcher = address(0);
         _endBatching(refund);
+        _batcher = address(0);
+    }
+
+    function lockCallback() external returns (address caller) {
```

Comment:
Nit: 
```suggestion
    /// @inheritdoc IGateway
    function lockCallback() external returns (address caller) {
```

Reply:
Really like your attention to doc stuff! Thanks